src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -43,9 +43,11 @@ class MarathonModule(conf: MarathonConf, http: HttpConf, zk: ZooKeeperClient)
   val log = Logger.getLogger(getClass.getName)
 
   def configure() {
+
     bind(classOf[MarathonConf]).toInstance(conf)
     bind(classOf[HttpConf]).toInstance(http)
     bind(classOf[ZooKeeperClient]).toInstance(zk)
+    bind(classOf[MarathonSchedulerDriverHolder]).in(Scopes.SINGLETON)
```

Comment:
what do you mean by that? Maybe my comment above explained that (explaining the holder concept).